db/postgres/index.js

```diff
@@ -0,0 +1,34 @@
+const { Pool } = require('pg');
+
+const pg = new Pool({
+  user: 'keewooklee',
+  password: '',
+  port: 5432,
+  host: 'localhost',
+  database: 'reviewsdc',
+});
+
+pg.connect((err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log('connected to DB(postgres)');
+  }
+});
+
+pg.query(`CREATE TABLE IF NOT EXISTS reviews (
+  id SERIAL PRIMARY KEY,
+  product_id int,
+  rating int,
+  summary VARCHAR,
+  recommend boolean,
+  response VARCHAR,
+  body VARCHAR,
```

Comment:
Review Body - The review body will be a free-form multimedia input where the user can submit text and images regarding their experience with the product. 
The text submitted as part of the review will be between 50 and 1000 characters long.